tests/test_http_notebook.py

```diff
@@ -245,16 +215,10 @@ def test_http_notebook_08(self):
         self.assertNotIn('error', results['results'][0])
         self.assertNotIn('error', results['results'][1])
 
-        self.assertEqual(
+        self.assert_data_shape(
             results['results'][2],
             {
                 'kind': 'data',
-                'data': [
-                    'AAAAAAAAAAAAAAAAAAABBQ==',
-                    'AgAAAAAAAAAAAAAAAAAAAQU=',
-                    'RAAAABIAAQAAAAgAAAAAAAAAAQ==',
```

Comment:
Obviously these tests are pretty horrific, but some of them are kind of valuable.
This one tests that the global is actually getting set. If it wasn't, different data would get returned.
One approach to updating these now would be to keep the switch to `assert_data_shape`, but just put in `str` for the 0th, 1st, 3rd component.
(so `[str, str, 'RAAAABIAAQAAAAgAAAAAAAAAAQ==', str]`), which would test the actual returned data but not really care much about the headers.

I think the actually right thing to do is have `edgedb-python` expose (undocumented) parsing/serialization functions that can work on `bytes`, but that shouldn't block this PR

Reply:
OK, changed to still test for data bytes.